tweet-api/service.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
I think that whole file needs to be simplified a lot because I see lots of extraneous things, but I can take care of it later 👍🏻

Reply:
@soulshake